lib/email.js

```diff
@@ -34,16 +45,16 @@ exports.sendConfirmationEmail = function(host, email, confirmation, userid, cb)
 exports.contactSeller = function(host, from, to, message, listingId, cb) {
   // setup e-mail data with unicode symbols
   var mailOptions = {
-      from: '"noreply" <noreply@tuskmarketplace.com>', // sender address
+      from: '"pleasereply" <pleasereply@tuskmarketplace.com>', // sender address
```

Comment:
Yes that is true. However we received feedback that noreply was confusing. Most email clients display that as the email address and it confuses people. This way people will click reply and the reply-to field will get populated. 

If people understood this as a reply-to that would be great but mail clients don’t display it that way unfortunately.
> On Sep 10, 2016, at 1:42 PM, Brian Cefali <notifications@github.com> wrote:
> 
> In lib/email.js <https://github.com/TuftsTusk/ringmaster/pull/18#discussion_r78280969>:
> 
> > @@ -34,16 +45,16 @@ exports.sendConfirmationEmail = function(host, email, confirmation, userid, cb)
> >  exports.contactSeller = function(host, from, to, message, listingId, cb) {
> >    // setup e-mail data with unicode symbols
> >    var mailOptions = {
> > -      from: '"noreply" <noreply@tuskmarketplace.com>', // sender address
> > +      from: '"pleasereply" <pleasereply@tuskmarketplace.com>', // sender address
> What is it for? Because int his case they aren't even replying to pleasereply@tmp.com <mailto:pleasereply@tmp.com>, but rather the email in replyto.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TuftsTusk/ringmaster/pull/18/files/a10bf340a47edfc004aa498629e4c1855a685aba#r78280969>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACtOnYqPVrED7xg3CcpwO6VgPCC3ER5Uks5qouvvgaJpZM4J4TmX>.
>